ios/FluentUI/People Picker/ContactCollectionViewCell.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+class ContactCollectionViewCell: UICollectionViewCell {
+    static var identifier: String { return "ContactCollectionViewCell" }
```

Comment:
static let identifier: String = "ContactCollectionViewCell"

Reply:
A cleaner way of writing it!